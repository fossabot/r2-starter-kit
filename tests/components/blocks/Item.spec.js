import React from 'react';
import { shallow } from 'enzyme';
import Component from 'components/blocks/Item';

describe('<Item />', () => {
  const initProps = {
    breadcrumb: {},
    content: {},
  };

  let wrapper;
  beforeAll(() => {
    wrapper = shallow(<Component {...initProps} />);
  });

  it('should render initial component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
