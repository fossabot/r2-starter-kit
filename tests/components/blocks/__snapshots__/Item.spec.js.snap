// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Item /> should render initial component 1`] = `
<div
  className="ts text container"
>
  <Breadcrumb
    handleClick={[Function]}
    title=""
  />
  <div
    className="ts divider"
  />
  <Content
    authorEmail=""
    authorName=""
    description=""
    image=""
    placeholder=""
    title=""
  />
</div>
`;
